events/elections/2017/adnanabdulhussein_bio.md

```diff
@@ -37,7 +37,7 @@ the Kubernetes community. I think it's important that we uphold these values
 when defining and practicing a Code of Coduct and refining the contributor
 experience.
 
-Another area of the Steering Commitee I'm interested in is furthering the
+Another area of the Steering Committee I'm interested in is furthering the
```

Comment:
This is a historical document. Please don't change this file.

Reply:
I see. Thanks.